content/getting_started/3d_app_development/en.md

```diff
@@ -14,9 +14,6 @@ The Beta version of Zwin is still incomplete and has the following caveats relat
   - In particular, apps such as VSCode will not work.
   - Instead, **use a CUI editor such as Vim or nano**.
 
-- Some functions of the Wayland client do not work.
-  - **Pop-up windows do not open when right-clicking, etc.**
-
```

Comment:
Some functions of the Wayland client (such as relative pointer) still do not work, but they are difficult to describe and don't seem to matter so much, so I just removed these lines.